samplesheets/rendering.py

```diff
@@ -854,17 +856,23 @@ def get_study_tables(self, study):
         cache_backend = get_backend_api('sodar_cache')
         item_name = STUDY_TABLE_CACHE_ITEM.format(study=study.sodar_uuid)
         project = study.get_project()
-        # Get cached tables
-        if cache_backend:
-            item = cache_backend.get_cache_item(
-                app_name=APP_NAME,
-                name=item_name,
-                project=project,
+        if settings.SHEETS_ENABLE_STUDY_TABLE_CACHE:
+            # Get cached tables
+            if cache_backend:
+                item = cache_backend.get_cache_item(
+                    app_name=APP_NAME,
+                    name=item_name,
+                    project=project,
+                )
+                if item and item.data:
+                    logger.debug('Returning cached study tables')
+                    return item.data
+                logger.debug('Cache item "{}" not set'.format(item_name))
+        else:
+            logger.debug(
+                'Study table cache disabled in settings. See STUDY_TABLE_CACHE_ITEM in settings.py. Returning new tables.'
```

Comment:
Line length over 80 characters :) Sadly black doesn't fix lines with strings, has to be done manually..